lib/nimble.phx.gen.template/addons/ex_vcr.ex

```diff
@@ -0,0 +1,73 @@
+defmodule Nimble.Phx.Gen.Template.Addons.ExVCR do
+  use Nimble.Phx.Gen.Template.Addon
+
+  @cassette_directory "test/support/fixtures/vcr_cassettes"
+
+  @impl true
+  def do_apply(%Project{} = project, _opts) do
```

Comment:
I have a question here about this variable `_opts`, how to deal with the unused variable? if we don't use it should we rename it to `_` ? 🤔

Reply:
For unused variables, we can name with underscore or prefix underscore with the name. In this project, we set Credo to always name unused variables with an underscore for clarification.

Reply:
thanks @hanam1ni !